source/agora/test/LocalTransactions.d

```diff
@@ -73,9 +73,10 @@ unittest
     // Create 6 transactions - one for each validator
     auto txs = blocks[0].spendable().map!(txb => txb.sign());
     // Distribute them to different clients
-    txs.takeExactly(6).enumerate.each!((idx, tx) =>
-        network.clients[idx % network.clients.length]
-            .putTransaction(tx));
+    txs.takeExactly(6).enumerate.each!(
+        (idx, tx) {
+            network.ensureTxInPool(tx, (idx % network.clients.length).only);
```

Comment:
This defeats the purpose of the test tho, no?
We want nodes to ask for the missing TXs.

Reply:
It only ensures the tx is in the pool of that specific client

Reply:
I think @omerfirmak has a point as it is also putting the tx to each.

Reply:
But [this ](https://github.com/bosagora/agora/pull/2367/files#diff-2cb6a9d42ca703fefdc67b1c1062e63c52c36cb15156b0e48d4cf36b9ace51e8R1195-R1199) also does a `putTransaction?`

Reply:
Or maybe not.

Reply:
@omerfirmak : But only on one index

Reply:
It is correct on closer inspection.